Remotion/Web/Development.WebTesting.IntegrationTests/ScreenshotTest.cs

```diff
@@ -661,6 +666,32 @@ public void ScreenshotTransformationTest ()
       Helper.RunScreenshotTestExact<ScreenshotTest> (PrepareTest(), ScreenshotTestingType.Browser, test);
     }
 
+    private void RetryTest (Action action, int times)

```

Comment:
Do we want to make a //todo for removing this helper method once NUnit 3 gives us the option to retry test runs?

Reply:
Added todo comment